README.md

```diff
@@ -581,6 +581,49 @@ labels = ["LivesIn", "Visits"]
 path = "rel_examples.jsonl"
 ```
 
+#### spacy.LEMMA.v1
+
+The LEMMA task lemmatizes the provided text and updates the `lemma_` attribute in the doc's tokens accordingly.
+
+```ini
+[components.llm.task]
+@llm_tasks = "spacy.LEMMA.v1"
+examples = null
+```
+
+| Argument                  | Type                                    | Default                                                   | Description                                                                                                                                           |
+| ------------------------- | --------------------------------------- |-----------------------------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `template`                | `str`                                   | [lemma.jinja](./spacy_llm/tasks/templates/lemma.jinja) | Custom prompt template to send to LLM backend. Default templates for each task are located in the `spacy_llm/tasks/templates` directory.              |
+| `examples`                | `Optional[Callable[[], Iterable[Any]]]` | `None`                                                    | Optional function that generates examples for few-shot learning.                                                                                      |
+
+The LEMMA task implementation asks the LLM to annotate the passed text with double square brackets containing the corresponding
+lemmas, like this: for specific offsets, but simply expects a list of strings that represent the enties in the document.
```

Comment:
This is hard to read, perhaps it's easier to start with the example, because it's unclear upfront what the square brackets are used for. Also, typo "enties".

Reply:
This was messed up during merging, I think. Updated in https://github.com/explosion/spacy-llm/pull/164/commits/936d1495dd392d35b0cc79a5f4be99a8a6f1e5dc.